package.json

```diff
@@ -11,12 +11,17 @@
   "devDependencies": {
     "@storybook/addon-controls": "^6.3.7",
     "@storybook/addon-docs": "^6.3.7",
+    "@storybook/addon-postcss": "^2.0.0",
     "@storybook/server": "^6.3.7",
+    "autoprefixer": "^9",
     "jasmine-core": "~2.4.1",
     "karma": "~6.3.4",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coffee-preprocessor": "~1.0.1",
     "karma-jasmine": "~0.3.8",
+    "postcss": "^7",
```

Comment:
Hmm I think we can actually use PostCSS 8 now, and drop the `@tailwindcss/postcss7-compat` :tada:

FYI this is where I would look for clues:
https://dev.to/davidteren/getting-started-with-rails-6-1-tailwindcss-jit-webpacker-postcss-8-25g7
https://github.com/davidteren/tails_on_rails

Reply:
Or in this Gist: https://gist.github.com/leastbad/d4c7e1d7872a1e20c0e9a82340f2082f

Reply:
Will recreate a PR ;)